src/www/app/outline_server_repository/server.ts

```diff
@@ -103,6 +120,10 @@ export class OutlineServer implements Server {
   }
 
   checkReachable(): Promise<boolean> {
```

Comment:
Perhaps mention that this method should only be called on running servers.
As a side note, we need to better track and handle server state changes... That's hurting us on Electron.

Reply:
Yeah I noticed what was going on here was a bit... filmsy